src/BugsnagUnity/Payload/Exception.cs

```diff
@@ -141,8 +141,13 @@ public static Exception FromUnityLogMessage(UnityLogMessage logMessage, System.D
           {
             errorClass = match.Groups["errorClass"].Value;
             message = match.Groups["message"].Value.Trim();
-            lines = new StackTrace(logMessage.StackTrace, StackTraceFormat.AndroidJava).ToArray();
           }
+          else
+          {
+            errorClass = message;
```

Comment:
Can you add some level of commenting around the format of the AndroidJavaException message?